code/datums/components/crafting/recipes.dm

```diff
@@ -1767,30 +1767,14 @@
 	name = "Toiletbong"
 	category = CAT_ENTERTAINMENT
 	tool_behaviors = list(TOOL_WRENCH)
-	reqs = list(/obj/item/flamethrower = 1)
-	machinery = list( /obj/structure/toilet)
+	reqs = list(/obj/item/stack/sheet/iron = 1,
+			    /obj/item/flamethrower = 1,
+				/obj/item/tank/internals/plasma = 1)
```

Comment:
```suggestion
	reqs = list(/obj/item/stack/sheet/iron = 1,
			/obj/item/flamethrower = 1,
			/obj/item/tank/internals/plasma = 1)
```

Reply:
weird, it looks fine in VS Code
will attempt to fix it

Reply:
Done 👌